src/events/errors/commandError.ts

```diff
@@ -1,47 +1,43 @@
 import { Colors } from '@lib/types/constants/Constants';
 import { Events } from '@lib/types/Enums';
 import { LanguageKeys } from '@lib/types/namespaces/LanguageKeys';
-import { codeBlock } from '@sapphire/utilities';
+import { Awaited, codeBlock, isThenable } from '@sapphire/utilities';
 import { rootFolder } from '@utils/constants';
+import { cast } from '@utils/util';
 import { DiscordAPIError, HTTPError, MessageEmbed } from 'discord.js';
 import { Command, Event, KlasaMessage } from 'klasa';
 
-const BLACKLISTED_CODES = [
+const ignoredCodes = [
 	// Unknown Channel
 	10003,
 	// Unknown Message
 	10008
 ];
 
 export default class extends Event {
-	public async run(message: KlasaMessage, command: Command, _: string[], error: string | Error) {
+	public async run(message: KlasaMessage, command: Command, _: string[], error: Awaited<string> | Error) {
+		// Verify if the error is promise or not, if it is then await that promise
+		if (isThenable(error)) error = await error;
```

Comment:
*ironic, this doesn't seem to work*

Reply:
What do you mean?

Reply:
Fav showed in the dev server, throwing without await returned `[object promise]`